app/models/service_template_transformation_plan_task.rb

```diff
@@ -54,7 +54,7 @@ def task_active
   def preflight_check
     destination_cluster
     virtv2v_disks
-    network_mappings
+    network_mappings.each { |m| raise if m.ip_address.nil? }
```

Comment:
@fdupont-redhat it looks like most of the existing validation is done in the `calculate_abcd` methods, would be more consistent to do this check down here https://github.com/ManageIQ/manageiq/blob/cf83a9b59f2e5e0cae76cfdf16923ede9ad3f54f/app/models/service_template_transformation_plan_task.rb#L258